### PR TITLE
Service fabric sample with SF persistence

### PR DESCRIPTION
**Pull request by tmasternak**
Supersedes #2577

### CODE REVIEW

**Comment by tmasternak** [comment]
@Particular/docs-maintainers this is ready to merge.

File: samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs [hunk 1]
```diff
@@ -112,4 +131,32 @@ public void Abort()
         // Fire & Forget Close
         CloseAsync(CancellationToken.None);
     }
+}
+
+namespace NServiceBus.Persistence.ServiceFabric
```
Comment (SimonCropp) [inline samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs]: is the namespace required?

File: samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs [hunk 2]
```diff
@@ -112,4 +131,32 @@ public void Abort()
...
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    [DataContract(Namespace = "NServiceBus.Persistence.ServiceFabric", Name = "SagaEntry")]
+    sealed class SagaEntry : IExtensibleDataObject
```
Comment (SimonCropp) [inline samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs]: any reason this is sealed?

Comment (SimonCropp) [inline samples/azure/azure-service-fabric-routing/Core_6/CandidateVoteCount/EndpointCommunicationListener.cs]: can this be in a diff file?

**Comment by danielmarbach** [comment]
@SimonCropp addressed comments. Sealed introduced according to https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-work-with-reliable-collections

**Pull request merged**